docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device.rst

```diff
@@ -101,6 +101,11 @@ On platforms that natively support half-precision calculations (``bfloat16`` or
 of ``f32`` to achieve better performance (see the `Execution Mode Hint <#execution-mode-hint>`__).
 Thus, no special steps are required to run a model with ``bf16`` or ``f16`` inference precision.
 
+.. important::
+
+   The ``bf16`` floating-point precision appears to have some limitations that impact the
+   inference accuracy in LLM models. For more details, refer to this :ref:`article <limitation-of-the-bf16-inference-precision>`.
+
```

Comment:
`appears to have some limitations that impact the
   inference accuracy in LLM models` sounds vague, can we publish the actual accuracy drop of some well-known LLM models on well-known datasets (like https://github.com/ggerganov/llama.cpp?tab=readme-ov-file#quantization, which is more helpful for customer to be aware of the issue?

Reply:
We can keep the reference to the article, where more information will be included.